Initiate "guide" notebook for validating cell type assignments (#1001)
AlexsLemonade/OpenScPCA-analysis: analyses/cell-type-ewings/template_notebooks/README.md

1. `cnv-workflow`: This folder contains all template notebooks used in `cnv-annotation.sh`.
2. `auc-singler-workflow`: This folder contains all template notebooks used in `auc-singler-annotation.sh`.
3. `clustering-workflow`: This folder contains all template notebooks used in `evaluate-clusters.sh`. 

## Guide notebook for assigning and evaluating cell type annotations 

The `celltype-exploration.Rmd` notebook is meant to be used as a guide for assigning and evaluating the final cell type annotations for each library in `SCPCP000015`. 

> Instructions for using this guide: 

Reviewer: I think it would also be fine to say here "instructions are in the template notebook" instead of duplicating instructions, but that's up to you!

1. Ensure that you have a local copy of the results from `aucell-singler-annotation.sh`, `evaluate-clusters.sh` and `run-aucell-ews-signatures.sh` saved to `results`. 
2. Copy the contents of this notebook to a new notebook titled `<library_id>_celltype-exploration.Rmd` and save in `exploratory_analysis/final_annotation_notebooks`. 

> 3. Replace the `sample_id` and `library_id` with the correct IDs in the `params` list. 

Reviewer: and update the notebook title too in the yaml

4. Optionally, you may choose to update the choices for clustering based on the results from `evaluate-clusters.sh`. 

> All clusters used will be calculated with the Leiden algorithm and the modularity objective function. 

Reviewer: ```suggestion
All clusters used were calculated with the Leiden algorithm and the modularity objective function. 
```

Reviewer: since this notebook isn't actually performing clustering

> To modify the nearest neighbors (default: 20) and resolution (default: 0.5) chosen use the `cluster_nn` and `cluster_res` params. 

Reviewer: for some reason "chosen" and "use" next to each other threw me off, so suggesting this rephrase
```suggestion
To modify the nearest neighbors (default: 20) and resolution (default: 0.5) parameters use the `cluster_nn` and `cluster_res` params. 
```
